package.json

```diff
@@ -56,7 +56,7 @@
     "lodash": "4.17.21",
     "luxon": "^1.27.0",
     "method-override": "3.0",
-    "moment": "^2.24.0",
+    "moment": "^2.29.2",
```

Comment:
Annoying that bunyan still requires this. Did a quick look at  bunyan and it honestly looks the like repo may be effectively abandoned by the maintainer without giving direction to the community... It may be worthwhile for us to move to something like pino- https://github.com/pinojs/pino which is actively maintained.

Reply:
https://github.com/trentm/node-bunyan/issues/335#issuecomment-648583828 ⬅️ Last "plan" to maintain was 2020, last overall commit was 15mo ago...